CHANGELOG.md

```diff
@@ -5,19 +5,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 ## Unreleased
 
-### Fixed
-- getdefaultlocale deprecated
-
-## Unreleased
+### Added
+- Added `system caches list` and `system caches delete` commands which allow administrators to enumerate and delete R and Python runtime caches from Connect servers.
 
 ### Fixed
-
 - cacert read error when adding/updating a server.
-
-## Unreleased
-
-### Added
-- Added `system caches list` and `system caches delete` commands which allow administrators to enumerate and delete R and Python runtime caches from Connect servers.
+- getdefaultlocale deprecated
+- extra files not being included in write-manifest
```

Comment:
It would be helpful to add issue numbers and better descriptions to these entries.